bin/common.bash

```diff
@@ -228,3 +228,21 @@ check_openstack_credentials() {
         exit 1
     fi
 }
+
+# Checks what difference between the main repo and the sun repo and open a dialog if they differs.
+diff_check(){
+pushd "${here}/../" || exit
+
+result=$(git diff kubespray/)
+
+popd || exit
+
+if [[ $result ]]; then
+  echo -e "[\e[34mck8s\e[0m] The status of the kubespray git submodule differs from the expected status, either it is on another commit or there are file changes. This can cause unexpected versions to be installed or cause other errors. We recommend that you stop and check what has changed."
```

Comment:
Please use the log functions
```suggestion
  log_info "The status of the kubespray git submodule differs from the expected status, either it is on another commit or there are file changes. This can cause unexpected versions to be installed or cause other errors. We recommend that you stop and check what has changed."
```